rank/data/tests/works.ts

```diff
@@ -11,6 +9,13 @@ const tests: Test[] = [
       'Ensure that the query brings back the works we expect at the top of the list',
     eval: equalTo1,
     cases: [
+      {
+        query: 'information law',
+        ratings: ['zkg7xqm7'],
+        knownFailure: true,
+        description:
+          'Multi-word exact matches at the start of a title should be prioritised',
+      },
```

Comment:
Can you toss in a comment linking to the original Slack discussion and/or GitHub issue, so we know this test comes from a real user?